src/components/search_component/SearchComponent.js

```diff
@@ -10,17 +10,24 @@ export const SearchComponent = ({ handleChange }) => {
   };
 
   return (
-    <div className="searchBar">
+    <div className="searchContainer">
       <form>
         <input
+          className="searchInput"
           type="text"
           placeholder="artist name"
           value={artist}
           onChange={(e) => setArtist(e.target.value)}
```

Comment:
I like this approach where you sent only event.target.value to the function invocation. Only sending the data you need.